### PR TITLE
Compatibility with node 1.5.0

### PR DESCRIPTION
**Pull request by tuler**
Change configuration generated by deploy procedure to be compatible with node 1.5.0

### CODE REVIEW

File: apps/web/components/Deploy/SelfHosted/NodeConfig.tsx [hunk 1]
```diff
@@ -39,8 +39,19 @@ const inputBoxDeploymentBlockNumber: Record<number, number> = {
...
+const epochLengths: Record<number, number> = {
+    [arbitrum.id]: 43200, // arbitrum doesn't have a fixed block interval
+    [arbitrumSepolia.id]: 43200, // arbitrum doesn't have a fixed block interval
+    [base.id]: 43200, // 1 day on a 2s block time
+    [baseSepolia.id]: 43200, // 1 day on a 2s block time
+    [foundry.id]: 720, // 1 hour on a 5s block time
+    [mainnet.id]: 7200, // 1 day on a 12s block time
+    [optimism.id]: 43200, // 1 day on a 2s block time
+    [optimismSepolia.id]: 43200, // 1 day on a 2s block time
+    [sepolia.id]: 7200, // 1 day on a 12s block time
```
Comment (endersonmaia) [inline apps/web/components/Deploy/SelfHosted/NodeConfig.tsx:51]: Is there any expectations on 1 day for testnets 1 week for mainnet? Or this is completely up to the application developer?

**Review by endersonmaia — DISMISSED** [review]
LGTM

File: apps/web/components/Deploy/SelfHosted/NodeConfig.tsx (hunk 1 above)
Comment (tuler) [reply apps/web/components/Deploy/SelfHosted/NodeConfig.tsx:51]: Indeed. Maybe we could already hint 1 week default for mainnets.

Comment (tuler) [reply apps/web/components/Deploy/SelfHosted/NodeConfig.tsx]: Changed to 7 days on mainnets

Comment (endersonmaia) [reply apps/web/components/Deploy/SelfHosted/NodeConfig.tsx]: I suggest adding these values somewhere in the docs, not just as code comments.

At apps/docs/guide/running/running-application.md it's said that the default is `7200` but this value will be different according to the chain selected.

Comment (tuler) [reply apps/web/components/Deploy/SelfHosted/NodeConfig.tsx]: Running application is about Cartesi CLI.
Maybe somewhere under deployment

Comment (tuler) [reply apps/web/components/Deploy/SelfHosted/NodeConfig.tsx]: Maybe I can emphasize that I'm talking about the local CLI env in that section.
And then add this new information at https://docs.sunodo.io/guide/deploying/supported-networks

**Review by endersonmaia — APPROVED** [review]
LGTM

Don't forget to squash the fixups.

File: apps/web/components/Deploy/SelfHosted/NodeConfig.tsx (hunk 1 above)
Comment (tuler) [reply apps/web/components/Deploy/SelfHosted/NodeConfig.tsx]: Done

**Review by endersonmaia — APPROVED** [review]

**Pull request merged**